evm_arithmetization/src/fixed_recursive_verifier.rs

```diff
@@ -1548,7 +1449,7 @@ where
     /// single one. The combined proofs are segment proofs: they are proofs
     /// of some parts of one execution.
     ///
-    /// While regular root proofs can only assert validity of a
+    /// While regular segment proofs can only assert validity of a
```

Comment:
The rest of the comment isn't up to date anymore, since an aggregation can span multiple transactions.